compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaLexer.java

```diff
@@ -838,211 +838,211 @@ private boolean isHexIntFollowedIdentifier() {
+            case RETRY:
                 return getSyntaxToken(SyntaxKind.RETRY_KEYWORD);
-            case LexerTerminals.ROLLBACK:
+            case ROLLBACK:
                 return getSyntaxToken(SyntaxKind.ROLLBACK_KEYWORD);
-            case LexerTerminals.TRANSACTIONAL:
+            case TRANSACTIONAL:
                 return getSyntaxToken(SyntaxKind.TRANSACTIONAL_KEYWORD);
-            case LexerTerminals.ENUM:
+            case ENUM:
                 return getSyntaxToken(SyntaxKind.ENUM_KEYWORD);
-            case LexerTerminals.BASE16:
+            case BASE16:
                 return getSyntaxToken(SyntaxKind.BASE16_KEYWORD);
-            case LexerTerminals.BASE64:
+            case BASE64:
                 return getSyntaxToken(SyntaxKind.BASE64_KEYWORD);
-            case LexerTerminals.MATCH:
+            case MATCH:
                 return getSyntaxToken(SyntaxKind.MATCH_KEYWORD);
-            case LexerTerminals.CONFLICT:
+            case CONFLICT:
                 return getSyntaxToken(SyntaxKind.CONFLICT_KEYWORD);
 
-            case LexerTerminals.CLASS:
+            case CLASS:
                 return getSyntaxToken(SyntaxKind.CLASS_KEYWORD);
-            case LexerTerminals.CONFIGURABLE:
+            case CONFIGURABLE:
                 return getSyntaxToken(SyntaxKind.CONFIGURABLE_KEYWORD);
-            case LexerTerminals.WHERE:
+            case WHERE:
                 return getSyntaxToken(SyntaxKind.WHERE_KEYWORD);
-            case LexerTerminals.SELECT:
+            case SELECT:
                 return getSyntaxToken(SyntaxKind.SELECT_KEYWORD);
-            case LexerTerminals.LIMIT:
+            case LIMIT:
                 return getSyntaxToken(SyntaxKind.LIMIT_KEYWORD);
-            case LexerTerminals.OUTER:
+            case OUTER:
                 return getSyntaxToken(SyntaxKind.OUTER_KEYWORD);
-            case LexerTerminals.EQUALS:
+            case EQUALS:
                 return getSyntaxToken(SyntaxKind.EQUALS_KEYWORD);
-            case LexerTerminals.ORDER:
+            case ORDER:
                 return getSyntaxToken(SyntaxKind.ORDER_KEYWORD);
-            case LexerTerminals.BY:
+            case BY:
                 return getSyntaxToken(SyntaxKind.BY_KEYWORD);
-            case LexerTerminals.ASCENDING:
+            case ASCENDING:
                 return getSyntaxToken(SyntaxKind.ASCENDING_KEYWORD);
-            case LexerTerminals.DESCENDING:
+            case DESCENDING:
                 return getSyntaxToken(SyntaxKind.DESCENDING_KEYWORD);
-            case LexerTerminals.JOIN:
+            case JOIN:
                 return getSyntaxToken(SyntaxKind.JOIN_KEYWORD);
-            default:
-//                if (this.keywordModes.contains(KeywordMode.QUERY)) {
-//                    return getQueryCtxKeywordOrIdentifier(tokenText);
-//                }
+            case DEFAULT:
                 return getIdentifierToken();
         }
+        return getIdentifierToken();
+
```

Comment:
`case DEFAULT` is redundant right.
 Here and in the DocumentationLexer.
```suggestion
            default:
                return getIdentifierToken();
        }
```

Reply:
Corrected 6cd605d5ee542c4e79ff929a21dded10e960011e